minard/templates/calibdq_tellie.html

```diff
@@ -14,6 +14,13 @@ <h1>TELLIE Data Quality</h1>
     <a class="btn btn-outline-primary btn-lg" href="{{ url_for("calibdq_tellie", limit=limit, offset=offset-limit) }}">Back</a>
     <a class="btn btn-outline-primary btn-lg" href="{{ url_for("calibdq_tellie", limit=limit, offset=offset+limit) }}">Next</a>
     {% endif %}
+    Runs per page:
+    <select onchange="location=this.options[this.selectedIndex].value;">
+        <option value="{{url_for("calibdq_tellie", limit=10, offset=offset)}}">10</option>
+        <option value="{{url_for("calibdq_tellie", limit=25, offset=offset)}}">25</option>
+        <option value="{{url_for("calibdq_tellie", limit=50, offset=offset)}}">50</option>
+        <option value="{{url_for("calibdq_tellie", limit=100, offset=offset)}}">100</option>
+    </select>
```

Comment:
Currently if you select a different limit the page loads but the drop down menu always displays 10. You can have a look at the stream page code here: https://github.com/snoplus/minard/blob/master/minard/templates/snostream.html#L28 to see how to automatically have the dorrect limit selected.